src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -353,26 +355,25 @@ private void exportDescription(final URI uri) throws FcrepoOperationFailedExcept
 
     }
 
-    private List<URI> filterInboundReferences(final URI uri, final Model model) {
-        final List<URI> inboundMembers = new ArrayList<>();
+    private Set<URI> filterInboundReferences(final URI uri, final Model model) {
+        final String withSlash = withSlash(uri).toString();
+        final String withoutSlash = withoutSlash(uri).toString();
+        final Set<URI> inboundMembers = new HashSet<>();
         final List<Statement> removeList = new ArrayList<>();
-        for (final String p : config.getPredicates()) {
-            final StmtIterator inbound = model.listStatements(null, createProperty(p), (RDFNode)null);
-            while (inbound.hasNext()) {
-                final Statement s = inbound.next();
-                final String subject = s.getSubject().toString();
-                if (!subject.startsWith(uri.toString())) {
-                    removeList.add(s);
-                    inboundMembers.add(URI.create(subject));
-                }
+        for (final StmtIterator inbound = model.listStatements(); inbound.hasNext(); ) {
+            final Statement s = inbound.next();
+            final String subject = s.getSubject().toString();
+            if (!subject.equals(withSlash) && !subject.equals(withoutSlash)) {
+                removeList.add(s);
+                inboundMembers.add(URI.create(subject));
```

Comment:
@escowles Maybe we need a global inboundMembers list to keep track of those inbound resources that are already exported previously?